build.sbt

```diff
@@ -56,7 +55,7 @@ def baseSettings: Seq[Setting[_]] =
 //   minimalSettings ++ baseScalacOptions ++ Licensed.settings ++ Formatting.settings
 
 def addBaseSettingsAndTestDeps(p: Project): Project =
-  p.settings(baseSettings).configure(addTestDependencies)
+  p.settings(baseSettings).configure(addTestDependencies _)
```

Comment:
By-the-by, I'm pretty sure the underscores aren't required here and throughout. I think I know what happened: if you make a mistake, like forget a comma, the scala compiler is eager to tell how you should be explicitly eta-expanding these, but the error is actually elsewhere (i.e the missing comma).